aspnetcore/mvc/views/view-components.md

```diff
@@ -84,9 +84,15 @@ We recommend you name the view file *Default.cshtml* and use the *Views/Shared/C
 
 To use the view component, call `@Component.InvokeAsync("Name of view component", <anonymous type containing parameters>)` from a view. The parameters will be passed to the `InvokeAsync` method.  The `PriorityList` view component developed in the article is invoked from the *Views/Todo/Index.cshtml* view file. In the following, the `InvokeAsync` method is called with two parameters:
 
-```HTML
-@await Component.InvokeAsync("PriorityList", new { maxPriority = 2, isDone = false })
-   ```
+[!code-html[Main](view-components/sample/ViewCompFinal/Views/Todo/IndexFinal.cshtml?range=35)]
+
+## Invoking a view component as a Tag Helper
+
+For ASP.NET Core 1.1 and higher, you can invoke a view component as a [Tag Helper](xref:mvc/views/tag-helpers/intro):
+
+[!code-html[Main](view-components/sample/ViewCompFinal/Views/Todo/IndexTagHelper.cshtml?range=38-39)]
+
+Pascal-cased class and method parameters for tag helpers are translated into their [lower kebab case](http://stackoverflow.com/questions/11273282/whats-the-name-for-dash-separated-case/12273101#12273101). The Tag Helper to invoke a view component is `<vc>`. In the sample above, the `PriorityList` class becomes `priority-list`. The parameters to the view component are passed as arributes in lower kebab case.
```

Comment:
The wording "The Tag Helper to invoke a view component is `<vc>`" is perhaps a little confusing. This doesn't clearly explain that you prefix the lower-kebab cased view component name with `vc:`. I'm not sure off the top of my head what better wording is but I will come back to this if I think of something. Also, my understanding is that it's not the class name that gets lower kebob-ed, it's the view component name. The name could be different than the class but usually is the class name by convention.